spec/requests/admin/rvt_helpers_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe "Admin::RvtHelpers", type: :request do
+  before(:each) do
+    login_as(current_user)
+  end
+
+  describe "GET /index" do
+    context 'when the user has no privileges' do
+      let(:current_user) { create(:user) }
+
+      it 'is forbidden' do
+        get '/admin/rvt_helpers'
+
+        expect(response).to have_http_status(:forbidden)
+      end
+    end
+
+    context 'when the user is the rvt leader' do
+      let(:current_user) { Group.rvt.leader.user }
+
+      it 'is forbidden' do
```

Comment:
Should say 'is ok' or something like that

Reply:
Wow, I did not notice that you already check the changes, I fixed this typo.